proxy/http2/Http2ConnectionState.cc

```diff
@@ -575,6 +587,11 @@ rcv_ping_frame(Http2ConnectionState &cstate, const Http2Frame &frame)
 
   Http2StreamDebug(cstate.ua_session, stream_id, "Received PING frame");
 
+  if (cstate.get_zombie_event()) {
+    Warning("Ping frame for zombied sessoin %" PRId64, cstate.ua_session->connection_id());
```

Comment:
I thought you had said you were going to handle `PING` differently and reset the zombie timeout in that case.